### PR TITLE
feat: add dynamic firewall parameter

### PR DESCRIPTION
**Pull request by QuentinBtd**
I'm on fire 🔥 

Another option added

### CODE REVIEW

**Comment by QuentinBtd** [comment]
Hello @disc, I hope you are doing well !

Another small PR for you 👼

**Review by disc — APPROVED** [review]

**Comment by disc** [comment]
Hey @QuentinBtd, looks great!
Thanks for your contributions 👍

File: internal/pritunl/user.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ type User struct {
 	Gravatar        bool                     `json:"gravatar,omitempty"`
 	OtpAuth         bool                     `json:"otp_auth,omitempty"`
 	DeviceAuth      bool                     `json:"device_auth,omitempty"`
+	DynamicFirewall bool                     `json:"dynamic_firewall,omitempty"`
```
Comment (disc) [inline internal/pritunl/user.go:28]: Do you really need this property for the `pritunl_user` resource?

**Review by disc — CHANGES_REQUESTED** [review]

File: internal/pritunl/user.go (hunk 1 above)
Comment (disc) [reply internal/pritunl/user.go:28]: @QuentinBtd ^ minor changes requested

Comment (QuentinBtd) [reply internal/pritunl/user.go:28]: You saw it right, it's a mechanical addition on my part.
I checked the API calls to Pritunl, we can also remove `DeviceAuth` and `OtpAuth` from user resource

**Review by disc — APPROVED** [review]

**Pull request merged**